src/main/java/ru/l1ttleO/boyfriend/DelayedRunnable.java

```diff
@@ -0,0 +1,33 @@
+package ru.l1ttleO.boyfriend;
+
+import java.util.function.Consumer;
+import javax.annotation.Nullable;
+import org.jetbrains.annotations.NotNull;
+
+public class DelayedRunnable {
+
+    final public Thread thread;
+    final public long duration;
+    final public long startedAt;
```

Comment:
```suggestion
    public final Thread thread;
    public final long duration;
    public final long startedAt;
```
Reason: `Missorted modifiers`